Makefile

```diff
@@ -83,7 +83,8 @@ flake8:
 
 mypy:
 	@echo "Running mypy."
-	MYPYPATH=src $(PIPENV_CMD) run mypy --ignore-missing-imports --exclude pyspiffe/proto -p pyspiffe
+    # TODO: `--disable-error-code override` is a workaround to avoid an error with the protobuf generated code. Look for a better solution.
+	MYPYPATH=src $(PIPENV_CMD) run mypy --ignore-missing-imports --exclude pyspiffe/proto -p pyspiffe --disable-error-code override
```

Comment:
Shouldn't the exclude statement already take care of the generate code, or is this to suppress warnings for our code that calls the generated code? If the latter, it might be good to explicitly call that out in the comment.

Reply:
It is to suppress mypy errors in the generated code, that start showing up when make calls to it.

Reply:
Got it, thanks for the clarification!